docs/pages/docs/00-get-started/grakn-server.md

```diff
@@ -19,24 +19,22 @@ This guide offers advice on how to:
 
 Grakn releases from our [GitHub repository](https://github.com/graknlabs/grakn) are self-contained packages (tar.gz/zip) containing all the components you need to run Grakn.
 
-## Setting up Grakn
-
-### Minimum Requirements
+## Booting up Grakn
 
 Grakn can run on default Java settings (heap of 768MB, 1GB machine) if the knowledge graph is small enough.
 Recommended production settings are at least 4GB machine with 3GB heap.
 
-### Standalone Grakn
+Start Grakn with:
 
-You can start a standalone instance of Grakn by running `grakn server start`. This will produce a working environment for importing and analysing your data.
+```
+grakn server start
+```
 
-By default Grakn stores your data in the extracted directory, under `db/cassandra/` in the folder into which you unzipped the distribution zip.
+By default Grakn stores your data under `$GRAKN_HOME/db/` and log files under `$GRAKN_HOME/logs/`. It can be changed by updating the `data-dir` and `log.dirs` properties within `conf/grakn.properties`, respectively.
 
-We recommend changing this to another location to make upgrading Grakn easier and faster in the future. Having changed it, you need to specify the new location in the `conf/cassandra/cassandra.yaml` configuration file:
+For production use, we highly recommend changing them to an **external directory** outside of `$GRAKN_HOME` in order to make upgrading Grakn easier in the future.
```

Comment:
I guess we wanna emphasise this? as this sentence appeared twice in this section.

Reply:
Yes, the 2nd occurrence is to emphasize that.

Reply:
👍